azurelinuxagent/common/version.py

```diff
@@ -113,7 +113,7 @@ def get_distro():
 
 AGENT_NAME = "WALinuxAgent"
 AGENT_LONG_NAME = "Azure Linux Agent"
-AGENT_VERSION = '2.2.25.1'
+AGENT_VERSION = '2.2.26'
```

Comment:
Lost comment? New version is 2.2.27.

Reply:
At the time, your comment said "make it 2.2.26", so I did. Gosh, you're so picky. :-)